contrib/charts/navigator/values.yaml

```diff
@@ -28,7 +28,7 @@ controller:
     repository: jetstackexperimental/navigator-controller
     tag: latest
     pullPolicy: Always
-
+  logLevel: 100
```

Comment:
100 is *super* high. We'll get binary logs of every HTTP request with this. `4` is generally navigators own 'debug' level, 6 includes a log of HTTP requests (and their headers iirc) from all the underlying k8s clientsets etc.

Let's keep the default here as the actual default, `2` (or just omit it entirely if not specified in values) and then we can override it to 100 during our e2e tests (like you already are in this PR 😄)

Reply:
(although I still think 100 is *super* high for e2e tests even)

Reply:
Ok. I couldn't find any documentation about the log levels so I chose 100 since that's what the controller is currently hard coded to.